packages/compass-sidebar/src/components/sidebar.tsx

```diff
@@ -48,10 +53,18 @@ export function Sidebar({}: Props) {
   // TODO: filter
   // TODO: create database
   // TODO: non genuine warning label
+
+  // For now the new sidebar is always light
+  const [theme] = useState<ThemeState>({
+    theme: Theme.Light,
+  });
```

Comment:
This is super not important for now I guess, but it seems to be that you can just completely remove the provider here and then the new sidebar will just correctly follow the user theme (if this COMPASS_LG_DARKMODE feature flag is set of course), right? Or am I missing something?

Reply:
Good point..